### ISSUE TITLE
Customization Miku style

### ISSUE THREAD

**Issue by JeePopuwu** [fast4x/RiMusic#1624]
### Description

As we already know, we have plenty of ways to customize the main player, but how about adding more 'color' to it? (The request I'm making is of little importance and is purely aesthetic, so there's no rush to do it).

### Suggested Solution

As some may know, there's a video of Hatsune Miku where she appears in a player. So, why not bring that customization to Rimusic?

![Screenshot_2024-04-24-16-11-30-369_com miui videoplayer](https://github.com/fast4x/RiMusic/assets/159836335/a7e309c2-e5f1-485e-a782-a8a842174075)

This is an example of the player in the animation (I don't want the layout of the app's buttons to be like this, haha). Here's what we're interested in:

**1. Heart style:**
![Picsart_24-04-24_16-00-48-814](https://github.com/fast4x/RiMusic/assets/159836335/cbb0186d-c07e-4b01-80fc-0d8f988199a0)

![Screenshot_2024-04-24-15-55-49-529_com miui videoplayer](https://github.com/fast4x/RiMusic/assets/159836335/e73e2546-1029-45a5-8545-71d76542f472)

This one that hasn't been added has a glitch style.
And

**2.Playback bar style**

![Picsart_24-04-24_16-06-09-976](https://github.com/fast4x/RiMusic/assets/159836335/a0a7a5c0-a2b2-49de-988b-b6b721797ca6)

I would also like to request a style that I'm not sure if it's possible at the moment in Rimusic, but it would be fantastic to see it.

![Screenshot_2024-04-24-15-56-03-886_com miui videoplayer](https://github.com/fast4x/RiMusic/assets/159836335/d8ffc625-3c4d-451d-8558-088191ddf19b)

Two colors at once in the player (quite ambitious, lol), and if possible, with a neon style. I'm not sure how it could be implemented since I don't know anything about programming. But in my ignorance, I would propose that the two most prominent colors of the cover art be used.

### Alternatives

One of the ways that can be implemented (I think) is by adding a new option for the like/heart.

Here's an example of it.

![Picsart_24-04-24_16-26-36-078](https://github.com/fast4x/RiMusic/assets/159836335/a084bf46-b057-4a48-b9a2-d3106cccd27b)
![Picsart_24-04-24_16-34-06-171](https://github.com/fast4x/RiMusic/assets/159836335/985286b8-df24-4597-a879-e6a0ca2be60c)

### Additional Context

This is purely aesthetic, so it's not that important, and I know you're currently working on the foundation of Rimusic, so it's not necessary for the next update. Thank you in advance.

**Comment by JeePopuwu** [comment]
https://github.com/fast4x/RiMusic/assets/159836335/d108ee1f-91f1-4d67-8607-32bf7bde1fba

Here's an example of how it looks.

If you want Fast or Data-tron, they could also bring some of the colors from the animation.

**Comment by JeePopuwu** [comment]
![Screenshot_2024-04-24-15-55-20-107_com miui videoplayer](https://github.com/fast4x/RiMusic/assets/159836335/62fffff5-338c-44c7-a46e-43de7fa401e2)

Two notes I want to make:In the gradient heart, the player also has a gradient.AndA question that arose for me, and I'm not sure if it's possible, but would there be a possibility to allow certain songs to have certain customizations?Just as shown in the animation, where each song has its own customization.This request is somewhat similar to the guy who asked if leaving a certain speed on the player would remain that way.

**Comment by fast4x** [comment]
it's a nice feature, more elegant with animated, yes will be possible in player customizations. Miku it's a manga?

**Comment by dAtA-TRoN** [comment]
> Data-tron

I spy with my little eye... my name! Why, oh why, do I spy my name?

**Comment by NikunjKhangwal** [comment]
I keep seeing most random things here, it's actually hilarious 💀
>Miku it's a manga?

It's a mascot character of a software

**Comment by NikunjKhangwal** [comment]
> I thought Miku is a vocaloid.

Vocaloid itself is the name of software as far as i know. You can check wikipedia for more info

**Comment by JeePopuwu** [comment]
> Es una característica interesante, más elegante con animación, sí, será posible en las personalizaciones del jugador. ¿Miku ​​es un manga?

Miku is a Vocaloid character, which is also a software; there are many songs of her, and she was very popular in 2007.

I thought you already knew her x,D

**Comment by JeePopuwu** [comment]
> > Aquí hay un ejemplo de cómo se ve.
> > Si quieres Fast o Data-tron, también podrían traer algunos de los colores de la animación.
> 
> Veo con mi ojito... ¡mi nombre! ¿Por qué, ay por qué, veo mi nombre 🕵🏼‍♂️?

I don't remember well who was in charge of the visual stuff, I mentioned you because you always contribute ideas, so I also wanted your opinion.

**Comment by JeePopuwu** [comment]
> Sigo viendo la mayoría de cosas aleatorias aquí, en realidad es muy gracioso 💀
> 
> > ¿Miku ​​es un manga?
> 
> Es un personaje mascota de un software.

"Opinion about the ideas"

**Comment by dAtA-TRoN** [comment]
> I don't remember well who was in charge of the visual stuff, I mentioned you because you always contribute ideas, so I also wanted your opinion.

Haha, it's amusing how people seem to think I have some sort of authority here just because I talk a lot 😂. Well, hate to burst your bubble, but I'm just a tiny cog in the machine myself 🙂‍↔️...

![proxy-image3-ezgif.com-speed.gif](https://github.com/fast4x/RiMusic/assets/71184627/4449e342-95ee-483f-821e-e91295d40ac5)

**Comment by fast4x** [comment]
In this project everyone is free to say their own idea. It is obvious that if a user speaks more than others he will have a greater visibility in the eyes of others.

The sharing of ideas is a wealth in this project. Both those who express it and those who comment on it.

I am simply the project owner, you are all the soul of the same project.
❤️❤️❤️❤️❤️

**Comment by NikunjKhangwal** [comment]
You're the nicest developer I've seen here honestly. You too deserve all the praise 👏🏼

**Comment by dAtA-TRoN** [comment]
> You're the nicest developer I've seen here honestly. You too deserve all the praise 👏🏼

True. You know, all the developers I've met before here on GitHub were kind of like this:

![proxy-image.jpg](https://github.com/fast4x/RiMusic/assets/71184627/af3b8f69-bd0c-4cb1-9b85-8d1947e09b6e)

**Comment by NikunjKhangwal** [comment]
I can relate. It's even worse for people who aren't very tech literate, they usually get rude replies for asking questions because people here think it should be common sense. RiMusic feels like home compared to everything else.

**Comment by fast4x** [comment]
Where can i download animations?

**Comment by JeePopuwu** [comment]
> Where can i download animations?

I don't know x,,D

**Comment by JeePopuwu** [comment]
> Started... ![Screenshot_20240529_001232](https://private-user-images.githubusercontent.com/1484476/334589087-69166a23-805b-4afe-80c0-ea11f376edf9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5Mzc2MzIsIm5iZiI6MTcxNjkzNzMzMiwicGF0aCI6Ii8xNDg0NDc2LzMzNDU4OTA4Ny02OTE2NmEyMy04MDViLTRhZmUtODBjMC1lYTExZjM3NmVkZjkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjhUMjMwMjEyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDZlMjhhYjM2YTY3MWNmYzIxNzQxMWY2NmFjM2Y0MTc4NTBkZWNjM2RmOTY5MGFiMjJmNjJhODRmYWFhYjE5YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.9ekSSj0Qo-VirGH-omW8GRxlOAWbby_E5XXBxH9ilnk)

THANK YOU FOR EVERYTHING
